Makefile

```diff
@@ -22,9 +22,16 @@ endif
 VERSION_FILE := ./VERSION
 VERSION := $(shell grep -v ^\# $(VERSION_FILE))
 COMMIT_NO := $(shell git rev-parse HEAD 2> /dev/null || true)
+COMMIT_NO_SHORT := $(shell git rev-parse --short HEAD 2> /dev/null || true)
 COMMIT := $(if $(shell git status --porcelain --untracked-files=no),${COMMIT_NO}-dirty,${COMMIT_NO})
 VERSION_COMMIT := $(if $(COMMIT),$(VERSION)-$(COMMIT),$(VERSION))
 
+# args for building agent image
+BUILDARGS := $(if $(http_proxy), --build-arg http_proxy=$(http_proxy))
+BUILDARGS += $(if $(https_proxy), --build-arg https_proxy=$(https_proxy))
+AGENT_IMAGE := kata-containers/agent
```

Comment:
please use kata-containers/agent-dev which is more clear.

Reply:
note: kata-containers is an invalid name on hub.docker.com. and I had just defensively registered the katacontainers (only) on hub.docker.com.

Reply:
I'll rename it to "katacontainers/agent-dev"